src/main/java/util/FileUtil.java

```diff
@@ -0,0 +1,25 @@
+package util;
+
+import org.testng.ITestResult;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileUtil {
+
+    /***
+     * Clear log file
+     */
+    public static void clearFile(String file) throws IOException {
+        File logFile = new File(file);
+        new FileWriter(logFile, false).close();
+    }
+
+    /***
+     * Autodelete of generated video
```

Comment:
This is not autodelete, you are specifying when to delete the filder

Reply:
fixed